MicroProfile-JWT-Auth/tck-arquillian-extension/pom.xml

```diff
@@ -0,0 +1,83 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS HEADER.
+
+Copyright (c) 2017 Payara Foundation and/or its affiliates. All rights reserved.
```

Comment:
years

Reply:
Good shout, I need to update all of the copyright headers.